### DISCUSSION TITLE
How to get GitHub Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by Olek-Baran** [community/community#36050] [Other Feature Feedback, Questions, & Ideas] [answered]
My name is Oleksandr, student for programming.
I tried several times to get Galaxy Brian badge.

**Comment by Anmol-Baranwal** [comment]
Once, you get two accepted answers in the discussion marked by another person, then you can get your first galaxy brain badge (default).

**Comment by tzztson** [comment, marked as answer]
Hi, Olek,

To get the galaxy-brain badge, you need to answer at least 2 discussions.

The important thing is to get the mark -  "Make as Answer" from poster.

For example, now, you posted the question, you can click "Make as Answer" to my answer.

Then If I answer another question and poster mark me, then I will get galaxy-brain badge.

That's all. :)

![image](https://user-images.githubusercontent.com/112578419/196519517-79d87c50-bc6b-4ddc-986d-4071bab18a92.png)

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by archmage-24** [comment]
Answer at least 2 discussions.
Take note, you cannot mark your own answer as the answer to your own question.
Answer questions asked by users in discussions and get marked.
For example, if you mark this answer as "Mark as Answer", then I'll get a point for Galaxy Brain.
Similarly, you will get a point when someone marks your answer as the answer in some discussion.
